controller/core/pattern_selector.py

```diff
@@ -1,8 +1,15 @@
 import asyncio
 import functools
+from re import findall
 import lpminimk3
 import time
+import serial
+import numpy as np
 
+DMX_ADDR = '/dev/tty.usbserial-EN356968'
+DMX_START = bytearray([0x7E])
```

Comment:
Ha...this interesting! So 0x7E == 6 x '1' bits in a row == 104us @ 57600 baud which is a little more than the required 88us min break sequence followed by a single '0'. This looks like the start sequence but inverted? Shouldn't it be '10000001'? In any case, maybe leave a comment so we remember later.

Reply:
Hah, this is revealing my lack of knowledge in this area. Pulled the values from here: https://cdn.enttec.com/pdf/assets/70304/70304_DMX_USB_PRO_API.pdf

Reply:
Ah yes! This explains a lot. They totally have a different protocol for the USB serial side.